Passwords/twitter-banned.txt

```diff
@@ -126,7 +126,6 @@ diamond
 doctor
 doggie
 dolphin
-dolphins
 donald
```

Comment:
😢

Reply:
Note: You can't unban twitter banned passwords

> Developer's Note: Twitter doesn't sync its banned password lists here, otherwise LOL.

Reply:
Dolphins all the way down